cloudfoundry-client/src/main/java/org/cloudfoundry/client/v3/applications/ApplicationsV3.java

```diff
@@ -207,6 +207,14 @@ public interface ApplicationsV3 {
      */
     Mono<StartApplicationResponse> start(StartApplicationRequest request);
 
+    /**
+     * Makes the Restart Application request
+     *
+     * @param request the Restart Application request
+     * @return the response from the Restart Application request
+     */
+    Mono<RestartApplicationResponse> restart(RestartApplicationRequest request);
```

Comment:
Even though it makes sense to put "restart" next to "start" and "stop", the convention in CFJC is to keep method names in alphabetical order.